init_sd_card.scripts/init_sd_card.sh

```diff
@@ -575,8 +583,8 @@ unmount_disks
 
 echo "Finished preparing SD card. Please remove it and insert into $HOST_NAME."
 
-echo "Wait for a minute, then visit the following URL: http://$HOST_NAME:9000/"
-echo "SSH access is also provided by: ssh $HOST_NAME.local [-i $IDENTITY_FILE]"
+# echo "Wait for a minute, then visit the following URL: http://$HOST_NAME:9000/"
```

Comment:
$HOST_NAME.local?

Reply:
We can have these messages more precisely in the dts shell part. 

For example it will be `ssh hostname` because we set the right config in ~/.ssh/config.

Reply:
instead, it will always be

http://hostname.local:9000/

Reply:
so should the docs be updated to $HOST_NAME.local ?